### ISSUE TITLE
question about "def  unsup_rcnn_reg_loss"

### ISSUE THREAD

**Issue by songbae** [microsoft/SoftTeacher#57]
```
with torch.no_grad():
            _, _scores = self.teacher.roi_head.simple_test_bboxes(
                teacher_feat,
                teacher_img_metas,
                aligned_proposals,
                None,
                rescale=False,
            )
            bg_score = torch.cat([_score[:, -1] for _score in _scores])
            assigned_label, _, _, _ = bbox_targets
            neg_inds = assigned_label==self.student.roi_head.bbox_head.num_classes
            bbox_targets[1][neg_inds] = bg_score[neg_inds].detach()
        loss = self.student.roi_head.bbox_head.loss(
            bbox_results["cls_score"],
            bbox_results["bbox_pred"],
            rois,
            *bbox_targets,
            reduction_override="none",
        )
```

bbox_targets[0=]labels
bbox_targets[1] =label_weights
bbox_targets[2]= bbox_targets
bbox_targets[3]= bbox_weights

Q1. why only use bg_score from teacher_model? how about pos_inds?

**Comment by MendelXu** [comment]
Yes, you can refer to the ablation study in the paper.
